certificate-policy.md

```diff
@@ -600,90 +599,117 @@ The CA SHALL disclose all Cross Certificates that identify the CA as the Subject
+All CAs SHALL shall specify the procedures for validating information and identity evidence in the CA CPS. 
 
 ## 4.2 Certificate application processing
 
 ### 4.2.1 Performing identification and authentication functions
-The certificate request MAY include all factual information about the Applicant to be included in the Certificate, and such additional information as is necessary for the CA to obtain from the Applicant in order to comply with these Requirements and the CA's Certificate Policy and/or Certification Practice Statement. In cases where the certificate request does not contain all the necessary information about the Applicant, the CA SHALL obtain the remaining information from the Applicant or, having obtained it from a reliable, independent, third-party data source, confirm it with the Applicant. The CA SHALL establish and follow a documented procedure for verifying all data requested for inclusion in the Certificate by the Applicant.
+All CAs SHALL establish and follow a documented procedure for verifying all data requested for inclusion in the Certificate by the Applicant.
 
-Applicant information MUST include, but not be limited to, at least one Fully-Qualified Domain Name or IP address to be included in the Certificate's SubjectAltName extension.
+For end entity Domain Validation SSL certificates and end entity Organizational Validation SSL certificates: 
 
-Section 6.3.2 limits the validity period of Subscriber Certificates. The CA MAY use the documents and data provided in Section 3.2 to verify certificate information, provide that the CA obtained the data or document from a source specified under Section 3.2 no more than thirty-nine (39) months prior to issuing the Certificate.
+- The Applicant information SHALL include at least one Fully-Qualified Domain Name to be included in the Certificate's SubjectAltName extension 
+- All Fully-Qualified Domain Names to be included in the Certificate's SubjectAltName extension SHALL be verified in accordance with Section 3.2 before issuance of the certificate
+- CAA records for DotGov and DotMil domains SHALL be checked prior to issuance of any certificate and the CA SHALL act in accordance to the rules in the CAA records if present.  The CA SHALL identify in Section 4.2 of the CPS the Issuer Domain Name(s) used for CAA records.  
 
-The CA SHALL develop, maintain, and implement documented procedures that identify and require additional verification activity for High Risk Certificate Requests prior to the Certificate's approval, as reasonably necessary to ensure that such requests are properly verified under these Requirements.
+The CA MAY use the documents and data provided in Section 3.2 to verify certificate information, provided that the CA obtained the data or document from a source specified under Section 3.2 no more than 825 days prior to issuing the Certificate.  
```

Comment:
I know we *can* reuse validation information for 825 days, but *should* we allow it? This could result in a certificate in use whose actual validation was performed up to 1629 days in the past. I think we should discuss a reasonable self-imposed cap on validation information, such as 90 days.

Reply:
removed authorization from previous "who can submit" comments. 

In this scenario, authorization can be domain authorization.  Reuse of data still applies to domain authorization. 

How about 825 days for the data to be used / reused, and the certificate validity expiration cannot extend past 825 days from the date of the identity validation?

Reply:
> How about 825 days for the data to be used / reused, and the certificate validity expiration cannot extend past 825 days from the date of the identity validation?

So if we perform technical validation on day 0, then we reuse that data 300 days later, a certificate issued by reusing that validation data can only be valid for up to 525 days?

If so, that sounds reasonable and addresses my concern about allowing the potential for certificates being in use 1,650 days after their initial validation. 

:+1: